### PR TITLE
Added links to the Red Hat Container Catalog 

### PR DESCRIPTION
**Pull request by dayleparker**
I've now updated the guide with links to the pod images in the Container Catalog after chatting with the SBR-CFME team to determine the best location to direct customers to.

Suyog, can you please let me know if you think the updates look OK? (Screenshot attached to bug)
Many thanks for reviewing!

https://bugzilla.redhat.com/show_bug.cgi?id=1470493

Cheers,
Dayle

### CODE REVIEW

File: doc-Installing_on_OpenShift_Container_Platform/topics/installation.adoc [hunk 1]
```diff
@@ -26,7 +26,18 @@ To successfully deploy a {product-title} appliance on OpenShift Container Platfo
 OpenShift Container Platform 3.5 is required for this installation. Red Hat has not tested this procedure with earlier versions of OpenShift Container Platform.
 ====
 
-The {product-title_short} deployment uses three files to create the appliance: `cfme-template.yaml`, which is the {product-title_short} template used for the deployment, and `cfme-pv-example.yaml` and `cfme-pv-app-example.yaml`, two pod volume files. OpenShift Container Platform 3.5 includes these files by default.
+The {product-title_short} deployment uses three files to create the appliance: `cfme-template.yaml`, which is the {product-title_short} template used for the deployment, and `cfme-pv-example.yaml` and `cfme-pv-app-example.yaml`, two pod volume files. 
+
+These files create the OpenShift pods comprising the appliance, and are available from the Red Hat Container Catalog. To download the files to your system, follow the OpenShift instructions on the *Get this image* page for each of the following images:
```
Comment (suyogsainkar) [inline doc-Installing_on_OpenShift_Container_Platform/topics/installation.adoc]: @dayleparker The updates look great and the instructions are easy to understand. Though I was just wondering if we should make this step a bit more explicit like in a procedure, something along the lines of  (sorry to nitpick!) -
"To download the files to your system, click the *Get this image* tab? and select *OpenShift* from the *Choose your platform:* list, then follow the instructions on the page for each image:"  
What do you think?

Comment (dayleparker) [inline doc-Installing_on_OpenShift_Container_Platform/topics/installation.adoc]: Thanks for the suggestion, Suyog -- I think it's a good idea to be a little more clear. I've edited this sentence to better reflect the procedure (and language) on the Container Catalog pages. Please let me know your thoughts. :)

**Comment by dayleparker** [comment]
@suyogsainkar Hi Suyog,
Good suggestion (not sure where my previous comment went). I've edited the sentence about pulling the container images to be more clear -- please let me know what you think.
Cheers,
Dayle

**Comment by suyogsainkar** [comment]
@dayleparker Thanks for editing that sentence, Dayle. It looks great; merging to master now.

**Pull request merged**